src/client/app/components/admin/UserViewComponent.tsx

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable no-mixed-spaces-and-tabs */
```

Comment:
Same question from other file on how relates to other PR. Also applies to UsersDetailsComponent.